### PR TITLE
[PWN-5380] network selector

### PR DESCRIPTION
**Pull request by agroznykh**
🚀 [ JIRA TICKET ](https://p2pvalidator.atlassian.net/browse/PWN-5380)

### CODE REVIEW

File: packages/ui/src/styles/themes/dark.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ const theme = {
       primary: '#FFFFFF',
       secondary: '#F6F6F8',
       tertiary: '#FAFBFC',
+      special: '#FAFAFC',
```
Comment (beautyfree) [inline packages/ui/src/styles/themes/dark.ts:9]: Should be commented that this color from ios design system

File: packages/ui/src/styles/themes/light.ts [hunk 2]
```diff
@@ -6,6 +6,7 @@ const theme = {
       primary: '#FFFFFF',
       secondary: '#F6F6F8',
       tertiary: '#FAFBFC',
+      special: '#FAFAFC',
```
Comment (beautyfree) [inline packages/ui/src/styles/themes/light.ts:9]: Should be commented that this color from ios design system

File: packages/web/src/new/scenes/Main/Send/NetworkSelect/NetworkSelect.tsx [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+        <NetworkView
+          network={viewModel.network}
+          token={getTokenByNetwork(viewModel.network)}
+          payingWallet={viewModel.payingWallet}
+          feeInfo={viewModel.feeInfo.value}
+        />
+      }
+    >
+      {cautionEl()}
+
```
Comment (beautyfree) [inline packages/web/src/new/scenes/Main/Send/NetworkSelect/NetworkSelect.tsx:123]: Why don't use `map` for `getSelectableNetworks`?

File: packages/web/src/new/services/TransactionHandler/TransactionHandler.ts [hunk 4]
```diff
@@ -251,9 +264,9 @@ export class TransactionHandler implements TransactionHandlerType {
         }
       },
       {
-        retries: 10,
-        minTimeout: 1000,
-        maxTimeout: 60000,
+        retries: 30,
+        minTimeout: 1_000,
+        maxTimeout: 2_000,
```
Comment (turisap) [inline packages/web/src/new/services/TransactionHandler/TransactionHandler.ts:269]: I never knew this syntax is valid)))

**Review by turisap — DISMISSED** [review]

**Review by turisap — DISMISSED** [review]

**Review by beautyfree — DISMISSED** [review]

**Pull request merged**